spec/acceptance/suites/default/00_basic_test_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'spec_helper_acceptance'
 
+TEST_RVM_VERSION = (ENV.key?('TEST_RVM_VERSION') ? ENV['TEST_RVM_VERSION'] : '2.5.8')
```

Comment:
_(Informational only; no need to change:)_ A simpler idiomatic equivalent:
```suggestion
TEST_RVM_VERSION  = ENV['TEST_RVM_VERSION'] || '2.5.8'
```